assets/styles/style.css

```diff
@@ -0,0 +1,149 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css");
+@import url("https://enigma-ni.netlify.app/src/css/enigma.css");
+
+@font-face {
+  font-family: "SAMAN";
+  src: url("/assets/fonts/SAMAN.eot");
+  src: url("/assets/fonts/SAMAN.eot") format("embedded-opentype"),
+    url("/assets/fonts/SAMAN.woff2") format("woff2"),
+    url("/assets/fonts/SAMAN.woff") format("woff"),
+    url("/assets/fonts/SAMAN.ttf") format("truetype"),
+    url("/assets/fonts/SAMAN.svg#SAMAN") format("svg");
+}
+
+header {
+  border-block-end: 1px solid var(--border-color);
+}
+
+.nav-brand {
+  font-family: "SAMAN";
+  font-size: 2rem;
+}
+
+footer {
+  border-block-start: 1px solid var(--border-color);
+}
+
+.sidebar {
+  inline-size: 15rem;
+}
+
+@media screen and (min-width: 40rem) {
+  .sidebar-visible-tablet-onwards {
+    display: block;
+  }
+}
+
+.home-grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(25rem, 1fr));
+  justify-items: center;
+  gap: 1.5rem;
+  margin-block-start: 1rem;
+}
+
+.home-grid-container .card-img {
+  max-inline-size: 20rem;
+  max-block-size: 25rem;
+}
+
+#auth-modal .modal-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: var(--spacing-16);
+  text-align: center;
+}
+
+#auth-modal .check-container {
+  color: var(--color-green-600);
+}
+
+/* Product Page */
+.product-container {
+  display: grid;
+  gap: 1.5rem;
+}
+
+.product-filter {
+  grid-area: "filters";
+}
+
+.product-gallery {
+  grid-area: "products";
+  padding: 1rem;
+}
+
+.product-filter-container {
+  display: grid;
+  gap: 2rem;
+  padding: 1rem;
+}
+
+@media screen and (min-width: 80rem) {
+  /* desktop and above */
+  .product-container {
+    display: grid;
+    grid-template-areas: "filters products";
+    grid-template-columns: 20rem auto;
+  }
+}
+
+.product-filter-container .product-filter-heading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-filter-container .product-filter-options {
+  display: grid;
+  gap: 1.5rem;
+}
+
+.product-filter-container .product-filter-options > div {
+  display: grid;
+  gap: 0.5rem;
+}
+
+.product-gallery-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 15rem);
+  justify-content: space-evenly;
+  gap: 1.5rem;
+}
+
+/* Cart */
+.cart-container {
+  display: grid;
+  padding: 1rem;
+  gap: 1rem;
+}
+
+@media screen and (min-width: 64rem) {
```

Comment:
You can keep all media queries in one place either in separate file or in the same file with grouping

Reply:
iv kept related media queries together. so that its easy to find stuff. The reason I don't prefer keeping separate files is because each file creates a separate XHR call as a result increasing the page load time.